### DISCUSSION TITLE
Minor error in youtube video[6:15:40] (Gradients of loss or parameters)

### DISCUSSION THREAD

**Discussion by himani007** [mrdbourke/pytorch-deep-learning#1244] [Q&A]
Hey, 
I wanted to confirm if i am understanding this correct. It was mentioned in the youtube video on the time: 6:15:40 that Back propagation(4. loss backward) is moving backwards through the network to calculate the gradients if the parameters of our model w.r.t the loss. 

But as i crosschecked and always understood was that we calculate the gradients of the loss w.r.t to parameters and not visa versa. 

Please clear if this is a different thing you are referring to or am i right?

**Comment by apurvkumar023** [comment]
Well well welll I did read the maths behind it. what we actually do is
new_weight = old_weight - learning_rate*derivative of mean square error with respect to weight

and hat's how we get new weight.

**Reply by apurvkumar023** [reply]
and there is one more thing in which we sum the error to a node and then we try to minimise that but happens in big neural netwroks.

**Reply by himani007** [reply]
Well, thats wht i am sayijng, we take the derivative of the loss and not the parameters, but its said opposite in the youtube video. Must be a mistake!
